cli/lib/kontena/machine/digital_ocean/node_provisioner.rb

```diff
@@ -47,7 +47,14 @@ def run!(opts)
           ShellSpinner "Waiting for node #{droplet.name.colorize(:cyan)} join to grid #{opts[:grid].colorize(:cyan)} " do
             sleep 2 until node = droplet_exists_in_grid?(opts[:grid], droplet)
           end
-          set_labels(node, ["region=#{opts[:region]}", "az=#{opts[:region]}"])
+          set_labels(
+            node,
+            [
+              "region=#{opts[:region]}",
+              "az=#{opts[:region]}",
+              "provider=digital_ocean"
```

Comment:
cli commands are named with `digitalocean`, maybe follow that naming throughout?